.github/workflows/pr-welcome-msg.yml

```diff
@@ -14,7 +14,7 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - name: Create comment
-        uses: peter-evans/create-or-update-comment@a35cf36e5301d70b76f316e867e7788a55a31dae
+        uses: peter-evans/create-or-update-comment@34c499dfd22c2a7abf115562285b849e7cd261cb
```

Comment:
Instead of hashes, why not using v4.0.0 directly which is the latest?

Reply:
good point. I would expect that tbh. not sure whether there was some historical reason that we needed some fix/functionality that has not been covered that time by an existing version. :shrug: 

@fernflower any idea?

Reply:
updated in leapp-repository and it's ok there. so updating here too